packages/ag-charts-community/src/chart/series/cartesian/lineSeries.ts

```diff
@@ -118,6 +138,8 @@ export class LineSeries extends CartesianSeries<LineContext> {
     @Validate(OPT_STRING)
     yName?: string = undefined;
 
+    errorBar?: ErrorBarDatum = undefined;
```

Comment:
You may need to assign an instance here, otherwise the options assignment code won't know how to construct it - once we extract to an enterprise module, the enterprise module intiialisation will take care of this step intrinsically though.

Reply:
If I assign this to a `new ErrorBarConfig()`, how will the `LineSeries` class know whether or not the `'errorBars'` property was given in the input JSON?